README.md

```diff
@@ -19,6 +19,8 @@ At the time of joining, the HR potrays the company like it is match made in Heav
 | 10 | Usually help half or whole financially for IT events and certifications?|
 | 11 | Can you describe the culture of the company?|
 | 12 | How easily the upper management is approachable for suggestion or queries?|
+| 13 | How joining your company will help me imrove my skills ?|
```

Comment:
Their is a spelling mistake, please fix that, and also, this question could be frame in a better way, currently this question is too direct.

something like this
> What are some of the challenges I will be facing in projects that will help push my boundaries?